### ISSUE TITLE
SCHEMA Markup

### ISSUE THREAD

**Issue by sirtastic** [entrity/tipsy-web-public#25]
[Documentation](https://schema.org/Recipe)
Scroll to bottom for example of recipe markup to see where things should go if anything below is unclear.  I've added the ones I could. Couldn't figure out how to add these:

# Need this added: #
```
<meta itemprop="interactionCount" content="UserComments:140" />
```
Needs to pull comment count into content="UserComments:#", can be added anywhere near comments section. 

# Need this added: #
itemprop="image"

Example:
```
<img itemprop="image" src="bananabread.jpg"  alt="Banana bread on a plate"  />
```

# Need this added: #
```
<meta itemprop="url" content="http://myrecipesite.com/pinapple.html" />
```

Content needs to be the recipe URL. Can be placed anywhere on drink page.

**Comment by entrity** [comment]
@sirtastic : I read on http://schema.org/UserComments the following:

>UserInteraction and its subtypes is an old way of talking about users interacting with pages. It is generally better to use Action-based vocabulary, alongside types such as Comment.

I can't find an example of how to use Comment and commentCount instead of interaction-based attributes. Can you?

**Comment by entrity** [comment]
@sirtastic : nm. Found what I needed at the superclass: https://schema.org/CreativeWork

**Comment by entrity** [comment]
commit dfc20d01f6acdfc3a2688f8639fbf6de5ae3fff7
Author: Markham Anderson <quinesquines@gmail.com>
Date:   Sun Aug 2 11:04:21 2015 -0700

    Add schema.org meta. Closes #25

**Comment by entrity** [comment]
@sirtastic : Looks like you were implementing this. I've got to do a merge conflict now.
